Update quickstart-create-workspace.md (#117167)
Azure/azure-content: articles/synapse-analytics/quickstart-create-workspace.md

1. In the **Basics** tab, give the workspace a unique name. We'll use **mysworkspace** in this document
1. You need an ADLSGEN2 account to create a workspace. The simplest choice is to create a new one. If you want to re-use an existing one you'll need to perform some additional configuration. 
1. OPTION 1 Creating a new ADLSGEN2 account 
    1. Under **Select Data Lake Storage Gen 2 / Account Name**, click **Create New** and provide a global unique name, such as **contosolake**.
    1. Under **Select Data Lake Storage Gen 2 / File system name**, click **File System** and name it **users**.

> 1. OPTION 2 See the [**Prepare a Storage Account**](/azure/synapse-analytics/quickstart-create-workspace#prepare-an-existing-storage-account-for-use-with-azure-synapse-analytics) instructions at the bottom of this document.

Reviewer: ```suggestion
1. OPTION 2 See the [**Prepare a Storage Account**](#prepare-an-existing-storage-account-for-use-with-azure-synapse-analytics) instructions at the bottom of this document.
```

>     1. Under **Select Data Lake Storage Gen 2 / Account Name**, click **Create New** and provide a global unique name to it e.g. **contosolake**.
>     1. Under **Select Data Lake Storage Gen 2 / File system name**, click **File System** and name it **users**.
> 1. OPTION 2 See the [**Prepare a Storage Account**](https://learn.microsoft.com/en-us/azure/synapse-analytics/quickstart-create-workspace#prepare-an-existing-storage-account-for-use-with-azure-synapse-analytics) instructions at the bottom of this document.

Reviewer 2: ```suggestion
    1. Under **Select Data Lake Storage Gen 2 / Account Name**, click **Create New** and provide a global unique name, such as **contosolake**.
    1. Under **Select Data Lake Storage Gen 2 / File system name**, click **File System** and name it **users**.
1. OPTION 2 See the [**Prepare a Storage Account**](/azure/synapse-analytics/quickstart-create-workspace#prepare-an-existing-storage-account-for-use-with-azure-synapse-analytics) instructions at the bottom of this document.
```
Fixing text for MS style and fixing link to resolve warning
